pkg/node/manager/manager.go

```diff
@@ -282,7 +313,21 @@ func (m *manager) DeleteNode(nodeName string) error {
 // using custom networking
 func (m *manager) updateSubnetIfUsingENIConfig(cachedNode node.Node, k8sNode *v1.Node) error {
 	eniConfigName, isPresent := k8sNode.Labels[config.CustomNetworkingLabel]
-	if isPresent {
+	if isPresent || m.customNetworkEnabledInCNINode(k8sNode) {
+		if !isPresent {
```

Comment:
Since we're checking both isPresent and !isPresent, can we flatten these if statements into siblings, instead of nesting??

Reply:
I had to make some change here to work with bubbling up errors. But I feel nested if check in this case is easier to avoid repeat code.